DragonSlayer.cpp

```diff
@@ -1,20 +1,37 @@
 #include "DragonSlayer.h"
 #include "Dragon.h"
+#include "Utility.h"
 
-//DragonSlayer::DragonSlayer
+DragonSlayer::DragonSlayer(std::string name_, int hitPoints_, int armor_) : 
+    Character(hitPoints_ , armor_ , 4),
+    attackItem(new AttackItem),
```

Comment:
You can initialize your `attackItem` where it is declared.
Try to avoid the constructor member initializer list when you can and construct all of your member variables in-class
- [ ] revise

Reply:
You didn't resolve this.
- [ ] revise